README.md

```diff
@@ -1,6 +1,6 @@
 # agent-status-change
 
-[ ![Download](https://api.bintray.com/packages/hmrc/releases/agent-status-change/images/download.svg) ](https://bintray.com/hmrc/releases/agent-status-change/_latestVersion)
+A backend microservice responsible for handling the suspension status of agents using the Agent Services Account.
 
```

Comment:
it also handles terminating an agent (removing everything to do with the agent from collections in AS). fyi - suspensions is not used (don't think terminations is either), instead we rely on the ETMP agent record.

Reply:
okay, I can add this info. Thanks